uint16 id
uint8 cmd_mode
uint16 cmd_data
float32 pos_estimate
float32 vel_estimate
float32  torque_estimate
float32 temp